server_tests/test/clientTests.js

```diff
@@ -1248,53 +1250,31 @@ describe("RUST API TEST SUITE", function() {
             });
         });
 
-        it('Expect cameraInExclusiveUse Error. camera._bublTimelapse cannot be run when another timelapse capture procedure is already active', function() {
+        it.only('Expect cameraInExclusiveUse Error. camera._bublTimelapse cannot be run when another timelapse capture procedure is already active', function() {
             this.timeout(timeoutValue * 2);
-            var firstTimelapseDone = false;
-            var startTime = Date.now();
-            var timeSendAnother = 1000;
-            var stopTime = 5000;
-            var stopSent = false;
-            var stopErr;
+            var expectedResults = {
+              results: {
+                  _bublFileUris: ['']
+              }
+            };
+            var commandId;
+            var deferred = Q.defer();
 
-            //instantiate the first camera._bublTimelapse command
-            return testClient.bublTimelapse(sessionId, function(res) {
-                if(Date.now() - startTime >= stopTime && !stopSent) {
-                    stopSent = true;
-                    testClient.bublStop(res.body.id)
-                    .then(function(res) {
-                        stopErr = Comparison.bublStopOutput(res);
-                    });
+            return Q.all([testClient.bublTimelapse(sessionId, function onUpdate (res) {
+              if (!commandId) {
+                commandId = res.body.id;
+
+                  return testClient.bublTimelapse(sessionId)
+                  .then(
+                    () => assert.fail('Should have received cameraInExclusiveUse'),
+                    (err) => Comparison.cameraInExclusiveUseError(err)
+                  )
+                  .then(() => testClient.bublStop(commandId), deferred.reject)
+                  .then((res) => Comparison.bublStopOutput(res), deferred.reject)
```

Comment:
(this may actually be incorrect and cause havok, depending on what `deferred.reject()` returns, which I assume is nothing since the documentation doesn't specify)